src/Components/App/App.js

```diff
@@ -1,27 +1,27 @@
 import './App.scss';
 import {Homepage} from '../HomePage/Homepage';
-import { Header } from '../Header/Header';
-import { Form } from '../Form/Form';
-import { Movie } from '../Movie/Movie';
-import { Music } from '../Music/Music';
-import { Podcast } from '../Podcast/Podcast';
-import { BoardGame } from '../BoardGame/BoardGame';
-import { CardGame } from '../CardGame/CardGame';
+import {Header} from '../Header/Header';
+import {Form} from '../Form/Form';
+import {Movie} from '../Movie/Movie';
+import {Music} from '../Music/Music';
+import {Podcast} from '../Podcast/Podcast';
+import {BoardGame} from '../BoardGame/BoardGame';
+import {CardGame} from '../CardGame/CardGame';
 import React, {Component} from 'react';
-import {Route} from 'react-router-dom';
-import { getAllMovies, getAllPodcasts, getAllCardGames, getAllMusic, getAllBoardGames} from '../../apiCalls.js'
-import { BrowsePage } from '../BrowsePage/BrowsePage'
-import { Footer } from '../Footer/Footer'
+import {Route, Switch} from 'react-router-dom';
+import {getAllMovies, getAllPodcasts, getAllCardGames, getAllMusic, getAllBoardGames} from '../../apiCalls.js'
+import {BrowsePage} from '../BrowsePage/BrowsePage'
+import {Footer} from '../Footer/Footer'
```

Comment:
I noticed that from lines 3 to 11, you removed the spaces in between the curly braces. My understanding is that we want those spaces for better readability.

Reply:
@Atos20  I thought so too, it's interesting that the linter doesn't like it.

Reply:
Yeah I need to adjust my linter, it defaults to remove the spaces here. Sorry about that